Code/Engine/GameEngine/GameApplication/Implementation/GameApplication.cpp

```diff
@@ -160,9 +161,10 @@ void ezGameApplication::Run_Present()
 
       ExecuteTakeScreenshot(pOutput, ctxt);
 
-      if (pWindowPlugin->GetWindow())
+      if (pWindowPlugin->GetWindow() && !bExecutedFrameCapture)
       {
         ExecuteFrameCapture(pWindowPlugin->GetWindow()->GetNativeWindowHandle(), ctxt);
+        bExecutedFrameCapture = true;
```

Comment:
Wouldn't you need to reset this somewhere? 

Afaik you can press a button to do a capture, this wouldn't work more than once then.

Reply:
The reset happens right before the for loop, in line 142:
```cpp
  bool bExecutedFrameCapture = false;
  for (ezActor* pActor : allActors)
  {
    //...
```